doc/WebRTC.xml

```diff
@@ -292,9 +295,8 @@
       <para>In case a connection is dropped the device shall reconnect automatically. Each client
         should use an individual ascending interval strategy to avoid that all clients reconnect at
         the same time.</para>
-      <para>Both device and client shall immediately send a register command as specified in section
-          <xref linkend="section_register"/> when they receive the HTTP 101 switching
-        protocol.</para>
+      <para>Both a client and a device must provide their JWT access token using the Authorization header
+        or via query parameter on the WebSocket URI as defined by [RFC6750] Section 2.3.</para>

```

Comment:
It is not as a general fact. The full URI is still completely encrypted within the TLS channel.
But usually, care needs to be taken by the implementers of the service to avoid leaking that token within their logs. But beyond the telemetry there should not be any issues.